lib/metamodelutil.js

```diff
@@ -168,6 +173,20 @@ function resolveTypeNames(metaModel, table) {
         metaModel.type.namespace = resolveName(name, table);
     }
         break;
+    case `${MetaModelNamespace}.ObjectMapKeyType`:
+    case `${MetaModelNamespace}.ObjectMapValueType`: {
+        metaModel.type.namespace = resolveName(metaModel.type.name, table);
```

Comment:
You don't include 

```
(metaModel.decorators || []).forEach((decorator) => {
            resolveTypeNames(decorator, table);
        });
```

See the other case. This means that any decorators that refer to types (yes, you can do that) won't have those types resolved for their decorators.